src/integrationTest/java/eu/xenit/gradle/testrunner/AbstractIntegrationTest.java

```diff
@@ -85,18 +89,25 @@ protected void testProjectFolderExpectFailure(Path projectFolder, String task, S
     }
 
     protected GradleRunner getGradleRunner(Path projectFolder, String task) throws IOException {
-        File tempExample = temporaryFolder.newFolder(projectFolder.getFileName().toString());
+        return getGradleRunner(projectFolder, task, "--stacktrace", "--rerun-tasks", "--info");
+    }
+
+    protected GradleRunner getGradleRunner(Path projectFolder, String task, String... additionalArguments) throws IOException {
+        File tempExample = getOrCreateTemporaryFolder(projectFolder.getFileName().toString());
         FileUtils.copyDirectory(projectFolder.toFile(), tempExample);
         File gitDir = tempExample.toPath().resolve("_git").toFile();
         if (gitDir.exists()) {
             FileUtils.moveDirectory(gitDir, tempExample.toPath().resolve(".git").toFile());
         }
 
+        List<String> arguments = new ArrayList<>();
+        arguments.add(task);
+        arguments.addAll(Arrays.asList(additionalArguments));
+
         GradleRunner runner =  GradleRunner.create()
                 .withProjectDir(tempExample)
-                .withArguments(task, "--stacktrace", "--rerun-tasks", "--info")
+                .withArguments(arguments)
```

Comment:
Can you make sure to always run with `--stacktrace` & `--info`? That will make build failures more clear in the test output.

Reply:
Yes, I can